src/e2e-test/features/pubsub/pubsubadditionalfeature/PubSubToPubSub.feature

```diff
@@ -226,7 +230,7 @@ Feature: PubSub Source - Verification of PubSub to PubSub successful data transf
     Then Deploy the pipeline
     Then Run the Pipeline in Runtime
     Then Wait for pipeline to be in status: "Running" with a timeout of 240 seconds
-    Then Publish the messages for text format
+    Then Publish the messages with schema
     Then Validate OUT record count is equal to IN record count
     And Stop the pipeline
-    Then Verify the pipeline status is "Stopped"
+    Then Verify the pipeline status is "Stopped"
```

Comment:
I don't see data validation step for any of the features, are we working on it or skipping it ?